### ISSUE TITLE
default retinanet model only reaches 0.246 mAP 

### ISSUE THREAD

**Issue by myopengit** [keras-team/keras-io#432]
The default model in the https://github.com/srihari-humbarwadi/datasets/releases/download/v0.1.0/data.zip has an mAP of only 0.246. Can you provide a well-trained model according to this tutorial https://keras.io/examples/vision/retinanet/ ?

**Comment by srihari-humbarwadi** [comment]
The provided model gives ~32 mAP using the post processing parameters mentioned in the paper. Please make sure you are using 0.05 as your score threshold as mentioned in the paper

**Comment by srihari-humbarwadi** [comment]
The optimal parameters are already set in the retinanet guide. Please follow the comments to train for ~13 epochs achieve decent performance

**Comment by myopengit** [comment]
@srihari-humbarwadi  Many thanks for the reply. The Retinanet (resnet50) in Detectron2 model zoo reaches 0.374 map at 13 epochs (0.387 at 37 epochs). The same image Augmentations are used (short edge 800, max size 1333). Do you have any ideas on how to improve the Keras model to reach a similar performance?

**Comment by sachinprasadhs** [comment]
Hi, 

Additionally,
1. You can try to modify the predefined aspect ratio for the bounding box in the class AnchorBox: as per your use case `aspect_ratios = [0.5, 1.0, 2.0]`.
2. If you have enough dataset, you try to increase the backbone other than `resnet50`, like resnet101 etc.
3. You can load the imagenet weights of respective backbone(resnet50 in your case) , which will boost the accuracy of the model.

Follow the updated tutorial here https://keras.io/examples/vision/retinanet/#implementing-anchor-generator
